src/components/welcome/WelcomeLogin/index.tsx

```diff
@@ -29,13 +32,17 @@ const WelcomeLogin = () => {
         </Typography>
```

Comment:
Should we also remove or change the text above?
It still says "Choose how you would like to create your Safe Account" or is it okay as you can choose one of the onboard wallets?

Reply:
I think its fine because there are multiple options in the onboard modal so the user is technically still choosing.